core/src/main/java/tech/pegasys/web3signer/core/metrics/vertx/PoolMetricsAdapter.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.core.metrics.vertx;
+
+import tech.pegasys.web3signer.core.metrics.Web3SignerMetricCategory;
+
+import io.vertx.core.spi.metrics.PoolMetrics;
+import org.hyperledger.besu.plugin.services.MetricsSystem;
+import org.hyperledger.besu.plugin.services.metrics.Counter;
+import org.hyperledger.besu.plugin.services.metrics.OperationTimer;
+import org.hyperledger.besu.plugin.services.metrics.OperationTimer.TimingContext;
+
+final class PoolMetricsAdapter implements PoolMetrics<TimingContext> {
```

Comment:
keeping it package scope deliberate?

Reply:
more that the besu version was package scoped and I copied. but not concerned if it package scoped or public. so i'll make public like everything else.